docs/download.md

```diff
@@ -0,0 +1,21 @@
+---
+id: download
+title: Apache Fury(incubating) Download
+---
+
+The official Apache Fury releases are provided as source artifacts.
+
+For binary install, please see Fury [install](/docs/start/install/) document.
+
+## The latest release 
+
+Apache Fury hasn't make a release since joined Apache Incubator.
+
+## Verify a release
+
+Please verify the release with corresponding hashes(sha512), signatures and [release KEYS](https://downloads.apache.org/incubator/fury/KEYS).
+
+
+## All archived releases
+
+Apache Fury hasn't make a release since joined Apache Incubator.
```

Comment:
Apache Fury (Incubating) hasn't made a release since joining the Apache Incubator.